### PR TITLE
Add Futureweb OG Private Domains

### PR DESCRIPTION
**Pull request by futureweb**
Customer level domains for Cloud CMS - Frontend arbitrary User Content & Backend Administration - Frontend + Backend Cookies - need seperation for security reasons

### CODE REVIEW

**Comment by futureweb** [comment]
- [x] DNS Authentication TXT entries added

**Comment by futureweb** [comment]
Are there any more information you need for merge?

File: public_suffix_list.dat [hunk 1]
```diff
@@ -11217,6 +11217,24 @@ fbxos.fr
...
 
+// Futureweb OG : http://www.futureweb.at
+// Submitted by Andreas Schnederle-Wagner <schnederle@futureweb.at>
+ex.ortsinfo.at
+at.ex.ortsinfo.at
+com.ex.ortsinfo.at
+de.ex.ortsinfo.at
+eu.ex.ortsinfo.at
+net.ex.ortsinfo.at
+tirol.ex.ortsinfo.at
```
Comment (sleevi) [inline public_suffix_list.dat]: Could you replace these with `*.ex.ortsinfo.at` ?

File: public_suffix_list.dat [hunk 2]
```diff
@@ -11217,6 +11217,24 @@ fbxos.fr
...
+futuremailing.at
+kunden.ortsinfo.at
+at.kunden.ortsinfo.at
+com.kunden.ortsinfo.at
+de.kunden.ortsinfo.at
+eu.kunden.ortsinfo.at
+net.kunden.ortsinfo.at
+tirol.kunden.ortsinfo.at
```
Comment (sleevi) [inline public_suffix_list.dat]: And these with `*.kunden.ortsinfo.at` ?

File: public_suffix_list.dat [hunk 3]
```diff
@@ -11217,6 +11217,24 @@ fbxos.fr
...
+// Futureweb OG : http://www.futureweb.at
+// Submitted by Andreas Schnederle-Wagner <schnederle@futureweb.at>
+ex.ortsinfo.at
+at.ex.ortsinfo.at
+com.ex.ortsinfo.at
+de.ex.ortsinfo.at
+eu.ex.ortsinfo.at
+net.ex.ortsinfo.at
+tirol.ex.ortsinfo.at
+futuremailing.at
```
Comment (sleevi) [inline public_suffix_list.dat]: The sorting here is incorrect, per https://github.com/publicsuffix/list/wiki/Guidelines

**Comment by futureweb** [comment]
@sleevi - wildcards should do the Job - also fixed sorting - thx 4 pointing it out - modified the pull request

**Pull request merged**